src/components/Kv/KvLightbox.vue

```diff
@@ -49,7 +49,12 @@
 					<div class="kv-lightbox__body"
 						ref="kvLightboxBody"
 					>
-						<slot>Lightbox body</slot>
+						<!-- gives lightbox content foundation grids the correct margins -->
+						<div class="row small-collapse">
+							<div class="small-12 columns">
```

Comment:
Since there's only a single column in this row, the column and row [can be combined](https://get.foundation/sites/docs/grid.html#combined-column-row), i.e.
```
<div class="row column collapse">
	<slot>Lightbox body</slot>
</div>
```

Reply:
Nice, thanks!

Reply:
Rats, for some reason the combination of `row column collapse` on the same div don't play well together.